### ISSUE TITLE
Support: Homogenized Universe Object

### ISSUE THREAD

**Issue by drewejohnson** [CORE-GATECH-GROUP/serpent-tools#11]
Implement a supporting object that will be used by the branching reader and results reader that stores data for a single homogenized universe at a single instance in time.

## Requirements
1. Should store the universe number, and, optionally, day, burnup, and burnup step for this instance. Optional because for a problem with no burnup, those values will not be present in the file.
1. Simple methods to add and get variable data, that will be sent from the readers.

## Optional
1. Output the data with some template engine like [Jinja2](http://jinja.pocoo.org/) or [Mako](http://www.makotemplates.org/)

## Holding up
#6
#7

**Comment by drewejohnson** [comment]
My opinion is that this class should have four public methods that access and update the data:
1. `set(variableName, variableValue, **kwargs)`
1. `get(variableName, **kwargs)`
1. `inf(variableName, **kwargs)`
1. `b1(variableName, **kwargs)`

## `set`
This method will store the data for some variable as `variableName` on the object's private data structure. We want to make this something that the user *shouldn't* directly alter, so maybe a warning should be thrown in the existing data is being overwritten, see [warning logs with python](https://docs.python.org/3.6/library/logging.html)

I added the `**kwargs` option because it may be convenient to the readers to add, say, the total broad group cross section and its associated uncertainty at the same time, i.e
`obj.set('infTotal', [t_0, t_1, t_2, ... t_n], unc=[u_0, u_1, u_2, ..., u_n])`
rather than
```
obj.set('infTotal', [t_0, t_1, t_2, ... t_n])
obj.set('infTotalUnc', [u_0, u_1, u_2, ..., u_n])
```
The names of the variables coming from the readers *should* be equivalent to those straight from SERPENT or converted to [camelCase](https://en.wikipedia.org/wiki/Camel_case) by the reader object.
The `set` method could take advantage of that and store infinite medium cross sections, critical buckling cross sections, and their associated uncertainties [if desired] in different data structures.

## `get`
This is the general method for obtaining any and all data from the object.
The `**kwargs` option could be used to return a default value if the desired value is not present (after raising a warning) rather than sending an error, or to return both a value and it's associated uncertainty.
Assuming the `infTotal` cross section and uncertainty have been loaded onto the object, the user could
```
obj.get('infTotal', unc=True)
# returns [t_0, t_1, t_2, ... t_n], [u_0, u_1, u_2, ..., u_n]
obj.get('absKeff`)
# returns only the value stored under 'absKeff'
```

## `inf` and `b1`
These methods would be shortcuts for accessing the cross sections, so that `obj.inf('total')` would be equivalent to `obj.get('infTotal')`.
Maybe not worth implementing at first release, because putting together the string passed into `obj.get` depends on if the variables are stored in `camelCase` or `SERPENT_STYLE`.

**Comment by drewejohnson** [comment]
Closed with #35
